api/src/index.js

```diff
@@ -11,21 +11,98 @@ app.use(bodyParser.json());
 
 const apiRouter = express.Router();
 
-// You can delete this route once you add your own routes
-apiRouter.get("/", async (req, res) => {
-  const SHOW_TABLES_QUERY =
-    process.env.DB_CLIENT === "pg"
-      ? "SELECT * FROM pg_catalog.pg_tables;"
-      : "SHOW TABLES;";
-  const tables = await knex.raw(SHOW_TABLES_QUERY);
-  res.json({ tables });
+//future-meals
+apiRouter.get("/future-meals", async (req, res) => {
+  try {
+    const futureMeals = await knex.raw("Select * From Meal WHERE Meal.when > NOW()")
+    if (futureMeals[0].length === 0) {
+    return res.json([]);
+    }
+    res.json(futureMeals[0]);
```

Comment:
This is just a small sidenote, but if `futureMeals[0].length === 0`, then that means that `futureMeals === []` anyway. 😄  This means that this `if` is not necessary, since once you are inside the `if`, the value of `futureMeals` is `[]` anyways. You can just return `futureMeals` without checking for length. 👍